sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/Docs/ConversationAnalysisClient.xml

```diff
@@ -78,56 +78,6 @@ JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
 Console.WriteLine(result.GetProperty("kind").ToString());
 ]]></code>
 </example>
-<remarks>
```

Comment:
So we're taking the schema out entirely? Shouldn't we at least point them to the REST API docs like line 84 below? How will the customer know what to type for anonymous/dynamic input? There's not even a direct link back to the REST API docs.

Reply:
Even just leaving the first few lines below this comment up to and including the REST API link would be helpful.